addOns/openapi/src/test/java/org/zaproxy/zap/extension/openapi/generators/HeadersGeneratorUnitTest.java

```diff
@@ -87,7 +87,7 @@ void shouldNotGenerateContentTypeIfRequestHaveNoMediaTypes() {
 
     @ParameterizedTest
     @ValueSource(strings = {"text/plain", "application/xml"})
-    void shouldNotGenerateContentTypeIfNotJsonNorWwwFormUrlEncoded(String mediaType) {
+    void shouldNotGenerateContentTypeIfNotJsonOrWwwFormUrlEncodedOrMultipart(String mediaType) {
```

Comment:
At this point might as well make it generic, e.g. `shouldNotGenerateContentTypeIfNotSupported`.